### DISCUSSION TITLE
Does multiple select work?

### DISCUSSION THREAD

**Discussion by ah0y** [woylie/flop_phoenix#268] [Q&A]
I'm trying to get multiple select working using something like this

```elixir
pet_types: [
  label: "Pet Types",
  type: "multiple_select",
  prompt: "",
  options: [
    {"dog", "dog"},
    {"cat", "cat"}
  ],
  op: :in
],
```

Based on the comment I found here https://github.com/woylie/flop_phoenix/discussions/189#discussioncomment-5757144 I think this should be possible? But in the browser I'm getting this weird search box:
<img width="262" alt="image" src="https://github.com/woylie/flop_phoenix/assets/25019088/f6fa4ad7-aee8-4d37-a4fc-dcb2778f6426">

I've tried different combinations of `op` like `:in`, `:contains`, and just omitting it, but am getting the same result.

**Comment by woylie** [comment]
As long as your input component supports a `multiple_select` type and accepts a list of values, it should work. The input component generated by default in Phoenix 1.7 is not set up that way, though. If you left it unchanged, you'd need to pass `type: "select", multiple: true`.

**Reply by ah0y** [reply]
That was what I was missing! Thank you for the quick reply!
